.github/ISSUE_TEMPLATE/prepare_release.md

```diff
@@ -14,14 +14,47 @@ For detailed info on the release process refer to https://github.com/waku-org/nw
  -->
 
 ### Items to complete
+
 - [ ] create release branch
 - [ ] assign release candidate tag
-- [ ] validate release candidate
 - [ ] generate and edit releases notes
-- [ ] open PR and merge release notes to master
-- [ ] cherry-pick release notes to the release branch
-- [ ] assign release tag to the cherry-picked release notes commit
-- [ ] create GitHub release
-- [ ] deploy the release to DockerHub
-- [ ] [deploy release to `waku.sandbox` fleet](https://ci.infra.status.im/job/nim-waku/job/deploy-waku-sandbox)
-- [ ] announce the release
+- [ ] _End user impact_: Summarize impact of changes on Status end users (can be a comment in this issue).
+- [ ] **validate release candidate**
+
+  - [ ] **On Waku fleets**
+    - [ ] Lock `waku.test` fleet to release candidate version
+    - [ ] Continuously stress `waku.test` fleet for a week (e.g. from `waku.sandbox`)
```

Comment:
> I think here it should be `waku-dev` machine instead of `waku.sandbox`
> 
> We sometimes called the `waku-dev` machine `waku-sandbox`, so it might generate confusions. But IMO better to stick to the `waku-dev` naming

oh yes good point! I agree that the "sandbox" term is confusing (I will stop using it as well to refer to that machine ;P)

We can mention `wakudev` or more specifically: `metal-01.he-eu-hel1.wakudev.misc`.